Makefile

```diff
@@ -2,5 +2,5 @@ update:
 	curl https://mkcert.org/generate/ | ./strip-non-ascii > certifi/cacert.pem
 
 publish:
-	python setup.py sdist bdist_wheel
+	pyproject-build
```

Comment:
maybe `python -m build`, as [the docs recommend](https://pypa-build.readthedocs.io/en/latest/)

Reply:
```suggestion
	python -m build
```